plugins/obs-qsv11/CMakeLists.txt

```diff
@@ -80,6 +80,11 @@ target_sources(
           QSV_Encoder_Internal.h
           bits/linux_defs.h
           bits/windows_defs.h)
+if(ENABLE_HEVC)
+  target_sources(
+    obs-qsv11 PRIVATE obs-qsv265.c QSV_HevcEncoder.h QSV_HevcEncoder_Internal.h
```

Comment:
Can you fold obs-qsv265.c into obs-qsv11.c to reduce copy/paste?

Reply:
I also think the QSV_HevcEncoder files should be folded into the original QSV_Encoder files.

Reply:
in progress..

Reply:
I merged changed as much as possible. Looks like further merging will lead to bad readability of the code.

Reply:
Ok, I'll take over from here.

Reply:
I combined the files. 700 line diff instead of 2000.